dist/voca.js

```diff
@@ -818,22 +818,27 @@
    * @memberOf Case
    * @param  {string} [subject=''] The string to convert to title case.
    * @param  {Array}  [noSplit]    Do not split words at the specified characters.
+   * @param  {boolean} [preserveUpperCase=false] preserve uppercases.
    * @return {string}              Returns the title case string.
    * @example
    * v.titleCase('learning to fly');
    * // => 'Learning To Fly'
    *
+   * v.titleCase('XMLHttpRequest', preserveUpperCase=true);
```

Comment:
Can you change `preserveUpperCase=true` to simple `true`? (to make the code runnable)